lib/generator/repository.rb

```diff
@@ -35,6 +35,10 @@ def create_tests_file
     def exercise_name
       @exercise_name ||= slug.tr('-', '_')
     end
+
+    def test_case_name
+      exercise_name + '_case'
```

Comment:
Yes it is.

Differences: 
* `exercise_name` is a property of the slug so it's less obvious where to put it.
* I didn't notice it was in the wrong place when I reviewed the PR that added it.

Similarities:
* They are both something that relate to Files/Filenames so should be under Generator::Files